apps/modernization-ui/src/apps/patient/add/extended/inputs/administrative/Administrative.tsx

```diff
@@ -0,0 +1,43 @@
+import { internalizeDate } from 'date';
+import { AdministrativeEntryFields } from 'apps/patient/profile/administrative/AdministrativeEntryFields';
+import { AdministrativeEntry } from 'apps/patient/profile/administrative/AdministrativeEntry';
+import { Heading } from 'components/heading';
+import styles from './Administrative.module.scss';
+import { FormProvider, useForm } from 'react-hook-form';
+import classNames from 'classnames';
+
+const defaultValue: AdministrativeEntry = {
+    asOf: internalizeDate(new Date()),
+    comment: ''
+};
+
+type Props = {
+    onChange: (data: AdministrativeEntry) => void;
+};
+
+export const Administrative = ({ onChange }: Props) => {
+    const form = useForm<AdministrativeEntry>({
```

Comment:
issue:  This is not a multi-valued area so it's part of the main form.  Only the multi-value entry areas require the sub-form.